src/Illuminate/Database/Eloquent/Factories/Factory.php

```diff
@@ -205,6 +205,22 @@ public function create($attributes = [], ?Model $parent = null)
         return $results;
     }
 
+    /**
+     * Create a collection of models and persist them to the database.
+     *
+     * @param array $attributes
+     *
+     * @return \Illuminate\Database\Eloquent\Collection|\Illuminate\Database\Eloquent\Model|mixed
+     */
+    public function createMany(array $records, ?Model $parent = null)
```

Comment:
Missing parent from phpdoc. phpdoc formatting is also incorrect,

Reply:
@GrahamCampbell 
```php
    /**
     * Create a collection of models and persist them to the database.
     *
     * @param  array  $records
     * @param  \Illuminate\Database\Eloquent\Model|null  $parent
     * @return \Illuminate\Database\Eloquent\Collection
     */
```
Does this look okay to you?

I want to change the description since it's the same as for the create method, but I'm not sure what a good description might be.